src/NServiceBus.RavenDB.AcceptanceTests/When_customizing_DocumentStore.cs

```diff
@@ -0,0 +1,121 @@
+﻿namespace NServiceBus.AcceptanceTests.Sagas
+{
+    using System;
+    using EndpointTemplates;
+    using AcceptanceTesting;
+    using NServiceBus.Configuration.AdvanceExtensibility;
+    using NServiceBus.RavenDB.Persistence;
+    using NServiceBus.Saga;
+    using NUnit.Framework;
+    using Raven.Client.Document;
+    using Raven.Client.Document.DTC;
+
+    public class When_customzing_DocStore : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void Should_honor_customized_values()
+        {
+            var context = new Context();
+            context.ExpectedResourceManagerId = Guid.NewGuid();
+
+            Scenario.Define(context)
+                .WithEndpoint<CustomizeDocStoreEndpoint>(b =>
+                {
+                    b.Given(bus => bus.SendLocal(new TestCmd { Name = "Doesn't matter, let's say Fred"}));
+
+                    b.CustomConfig(cfg =>
+                    {
+                        ConfigureRavenDBPersistence.GetDefaultPersistenceExtensions(cfg.GetSettings())
```

Comment:
Then it would be duplicating the call. I want to hang off the original call supplied globally to all tests in the Configure class. Special case since this is a Raven-specific test.